src/ast-to-react.js

```diff
@@ -121,6 +125,10 @@ exports.hastChildrenToReact = childrenToReact
  * @param {ReactBaseProps & ReactMarkdownProps & {depth: number, ordered: false}} props
  * @returns {ReactNode}
  *
+ * @callback VideoComponent
+ * @param {ReactBaseProps & ReactMarkdownProps & {url: string}} props
```

Comment:
`url` isn't a `<video>` attribute :thinking: 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#attributes

It may make sense to add attributes supported by the tag, like `src`, `autoplay`, `muted`, `poster`, and `loop`.

Reply:
Done!